apps/generator/test/integration.test.js

```diff
@@ -4,6 +4,8 @@
 
 const { readFile } = require('fs').promises;
 const path = require('path');
+const { promises: fsPromise } = require('fs');
+const { readFileSync} = require('fs');
```

Comment:
we talked during meeting is should not be like this
- 2 not needed duplicated requires, as there is one already in line 5
- you already in line 5 get access to `readFile` and now you want to also use the sync version

we need to be consistent and reuse what is already available
- so yes `readFile` always, with `await`
- extend `const { readFile } = require('fs').promises;` like `const { readFile, mkdir, writeFile } = require('fs').promises;`. `unlink` is not needed, as during meeting I said we do not need such a cleanup in test. `mkdir` probably not as well, because tbh I don't know why you create `await fsPromise.mkdir(outputDir, { recursive: true });` if generator does it anyways